### PR TITLE
Partial revert to commit in globalcontrol.sh

### PR DESCRIPTION
**Pull request by Skerse**
https://github.com/prasanthrangan/hyprdots/commit/732d7dc5d538a8bb045da6d36a0aced3113be76d changed a line that changed the way waybar (and possibly other configs) got there colors. For a while i noticed that waybar looked weird, like text was displaying red. Found that it was using my wallpaper for colors rather than the current gtk theme, looked and found the commit that changed that, i don't know if it was intentional but i think by default it should use the current gtk theme.

So tl;dr, this doesn't necessarily fix anything but just changes using wallpaper colors for the theme back to using the current gtk theme.

### CODE REVIEW

**Comment by akashgagda** [comment]
You can disable wallbash

**Comment by hyperupcall** [comment]
Thanks for fixing this saw your comment in https://github.com/prasanthrangan/hyprdots/pull/585#issuecomment-1851543911. Not sure how that slipped by, especially since I was modifying files with a find and replace regex...

**Comment by Skerse** [comment]
Yeah once again, no worries! Thanks for your work on making the script better too!

Also just now realized what wallbash was meant for.. (Thanks @akashgagda)

**Review by prasanthrangan — APPROVED** [review]
looks good, nice catch thanks!
